app/services/search_indexer.rb

```diff
@@ -164,9 +169,11 @@ def self.index(obj, force: false)
     end
 
     category_name = topic.category&.name if topic
+
     if topic
-      tags = topic.tags.select(:id, :name)
-      unless tags.empty?
+      tags = topic.tags.select(:id, :name).to_a
+
+      unless tags.blank?
```

Comment:
Can we do this instead?

```suggestion
      if tags.present?
```